examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/BasicClientFragment.kt

```diff
@@ -44,15 +52,23 @@ class BasicClientFragment : Fragment() {
       childFragmentManager.findFragmentById(R.id.addressUpdateFragment) as AddressUpdateFragment
 
     binding.writeNodeLabelBtn.setOnClickListener { scope.launch {
-      sendWriteNodeLabelAttribute()
+      val tlvWriter = TlvWriter()
```

Comment:
This seems somewhat inconvenient to maintain. We probably would want poj-to-tlv and tlv-to-poj (but that is reasonably complex I would imagine).

Maybe add a TODO/issue for this.

Reply:
Can you tell me what `poj-to-tlv` means?

Reply:
I added TODO comment.